lesson-4/style.css

```diff
@@ -40,33 +57,36 @@ h1 {
     text-align: center;
     padding: 20px;
 }
+
 .container-footer {
     background: #10367C;
     height: 630px;
     display: flex;
-    line-height: 45px;
     font-family: "Roboto", sans-serif;
     font-style: normal;
     font-size: 20px;
 
 }
+
 .footer-section {
     padding: 120px 100px;
```

Comment:
@AlKurlk its not good idea to set distance between columns using paddins/margins - set justify-content in footer-container